internal/kubeconfig/contexts.go

```diff
@@ -44,7 +44,7 @@ func (k *Kubeconfig) contextNode(name string) (*yaml.Node, error) {
 	return nil, errors.Errorf("context with name \"%s\" not found", name)
 }
 
-func (k *Kubeconfig) ContextNames() []string {
+func (k *Kubeconfig) ContextNames(filters map[string]string) []string {
```

Comment:
I'm open to suggestions how you want to handle optional parameters, be it a struct or separate function, or anything else